app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -270,8 +271,7 @@ private void takeScreenshotOfNotification(UiDevice device) {
     }
 
     private void takeScreenshotOfEraseSnackbar(UiDevice device) {
-                /* History Erase Notification */
-
+        TestHelper.webView.waitForExists(waitingTime);
```

Comment:
you could also wait for the progressbar to disappear:
 in testhelper:
    static UiObject progressBar = mDevice.findObject(new UiSelector()
            .resourceId("org.mozilla.focus.debug:id/progress")
            .enabled(true));

TestHelper.progressBar.waitForExists(webPageLoadwaitingTime);
Assert.assertTrue(TestHelper.progressBar.waitUntilGone(webPageLoadwaitingTime));

My pending PR has progressbar declaration.